src/platform_impl/windows/util.rs

```diff
@@ -193,7 +191,7 @@ pub fn adjust_window_rect_with_styles(
     status_map(|r| {
       *r = rect;
 
-      let b_menu: BOOL = GetMenu(hwnd).is_null().into();
+      let b_menu: BOOL = (GetMenu(hwnd).0 == 0).into();
```

Comment:
I don't know when did we change this but it should check for non-null. 
```suggestion
      let b_menu: BOOL = (GetMenu(hwnd).0 != 0).into();
```